release.sh

```diff
@@ -26,7 +31,13 @@ if ! diff -q <(git_remote_tags) <(git_local_tags) &>/dev/null; then
 fi
 
 # cut the next Flux release
-version=$(./gotool.sh github.com/influxdata/changelog nextver)
+version=$1
+if [ "$version" == "" ]; then
+    version=$(./gotool.sh github.com/influxdata/changelog nextver)
+fi
+if [[ $version != v* ]]; then
+    echo "Error: invalid tag specified. Tag must be prefixed with 'v'."
+    exit 1
+fi
```

Comment:
Should there be more validation to make sure that the version tag fits our convention? Or will it error if its not in a v0.01.01 format?

Reply:
In the case of a new major release, we won't need to mess with it. When one takes the wheel, however, there is a vector for human error. The `v` missing has been something that's bitten us a few times in other projects (`pkg-config` comes to mind), so I added validation there for that. Outside of that, when one overrides the default, it's okay to force that they own all the pieces.